docs/user/rstudio/ide/guide/documents/quarto-project.qmd

```diff
@@ -1,121 +1,182 @@
 
-![A section of the RStudio Options dialog with the R Markdown section selected in the sidebar. The Quarto tab is selected at the top. The body has 'Quarto' in bold, followed by 'Quarto is the next generation of R Markdown. This version of RStudio includes a preview of Quarto which you can enable below.' Below that, there is a radio button to Enable Quarto preview.](https://quarto.org/docs/tools/images/rstudio-enable-quarto.png){alt="A section of the RStudio Options dialog with the R Markdown section selected in the sidebar. The Quarto tab is selected at the top. The body has 'Quarto' in bold, followed by 'Quarto is the next generation of R Markdown. This version of RStudio includes a preview of Quarto which you can enable below.' Below that, there is a radio button to Enable Quarto preview."}
+To upgrade Quarto out of sync with the bundled version in RStudio, follow the directions at <https://quarto.org/docs/download/>
 
-If you have already installed the Quarto CLI then RStudio will detect this and enable Quarto features automatically. Alternatively, there is a version of Quarto built-in to RStudio that you can activate from R Markdown Preferences.\
+## Creating new documents
 
-### Creating Documents
+Use the **File** \> **New File** \> **Quarto Document** command to create new Quarto documents, which will launch the **New Quarto Document** wizard:
 
-Use the **File :** **New File : Quarto Document...** command to create new Quarto documents:
+![](https://quarto.org/docs/tools/images/new-quarto-doc.png){fig-alt="The 'New Quarto Document' dialog menu in RStudio."}
 
-![The 'New Quarto Document' dialog menu in RStudio.](https://quarto.org/docs/tools/images/new-quarto-doc.png){alt="The 'New Quarto Document' dialog menu in RStudio."}
-
-### Render and Preview
+## Render and Preview
 
 Use the **Render** button to preview documents as you edit them:
 
-![The top section of a qmd file as displayed in RStudio. There is a toolbar right above the document containing various options, including 'Render.' There is a stylized, segmented blue arrow pointing at the word.](https://quarto.org/docs/tools/images/rstudio-render.png){alt="The top section of a qmd file as displayed in RStudio. There is a toolbar right above the document containing various options, including 'Render.' There is a stylized, segmented blue arrow pointing at the word."}
+![](https://quarto.org/docs/tools/images/rstudio-render.png){fig-alt="The top section of a qmd file as displayed in RStudio. There is a toolbar right above the document containing various options, including 'Render.' There is a stylized, segmented blue arrow pointing at the word."}
 
 If you prefer to automatically render whenever you save you can check the **Render on Save** option on the editor toolbar.
 
-The preview will appear alongside the editor:
+The preview appears alongside the editor in the **Viewer** pane:
+
```

Comment:
This is true if **Preview in Viewer Pane** is selected which wasn't the default for me. 

<img width="392" alt="image" src="https://user-images.githubusercontent.com/114421526/195172570-51d9ec88-b2fb-45a0-8a5e-70c980e8f37b.png">